roles/openshift_logging/tasks/generate_jks.yaml

```diff
@@ -1,5 +1,39 @@
 ---
 # check if pod generated files exist -- if they all do don't run the pod
+- name: Check for valid external Elasticsearch hostnames
+  run_once: true
```

Comment:
Can you move the debug and set_stats into a `block` since they have the same when condition?
```yaml
- when: 
    - openshift_logging_es_allow_external | default(False) | bool
    - openshift_logging_es_hostname | default('') | regex_search('^[0-9]|[.][0-9]')
  block:
    - name: Check for valid external Elasticsearch hostnames
      run_once: true
      ....
    - debug:
        msg: ...
```